third_party/cares/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ list(APPEND CMAKE_ARGS
     "-DCARES_SHARED=OFF"
     "-DCARES_STATIC_PIC=ON"
     "-DCARES_BUILD_TOOLS=OFF"
+    "-DBUILD_SHARED_LIBS=OFF"
```

Comment:
If this is used, shouldn't we remove `-DCARES_STATIC=ON` and `-DCARES_SHARED=OFF` above?

Reply:
Good point :thinking: